polaris.shopify.com/content/patterns/date-picking/variants/date-range.md

```diff
@@ -323,8 +323,8 @@ function DateRangePicker() {
             )}
           </Box>
           <Box padding={{xs: 5}} maxWidth={mdDown ? '320px' : '516px'}>
-            <AlphaStack fullWidth gap="4">
-              <Inline>
+            <AlphaStack align="start" fullWidth gap="4">
```

Comment:
If this had the `fullWidth` property, then it shouldn't need `align="start"` and we can remove `fullWidth` now since that's the default

```suggestion
            <AlphaStack gap="4">
```

Reply:
Oops I guess I was too late. Maybe we can update this as part of https://github.com/Shopify/polaris/pull/8582